ftoolz/unsafe/reflection.py

```diff
@@ -0,0 +1,157 @@
+from typing import List, Set, Type, TypeVar
+
+T = TypeVar('T')
+
+
+def abstract(clz: Type[T]) -> bool:
+    """
+    Predicate that returns `True` iff given class has abstract methods.
+
+    >>> from abc import ABC, abstractmethod
+    >>> class A(ABC):
+    ...     @abstractmethod
+    ...     def test(self) -> None:
+    ...         pass
+    >>> abstract(A)
+    True
+
+    >>> class B(A):
+    ...     def test(self) -> None:
+    ...         pass
+    >>> abstract(B)
+    False
+    """
```

Comment:
I would add tests for empty abstract class
```python
    >>> class C(ABC):
    ...     pass
    >>> abstract(C)
    False
```

and for class that inherits from abstract, but does not implement abstract methods
```python
    >>> class D(A):
    ...     pass
    >>> abstract(D)
    True
```